test/integration/__tests__/live-api.js

```diff
@@ -487,8 +487,17 @@ describe('API integration', () => {
         // Successful cancel requests get ASSEMBLY_CANCELED even when it
         // completed, so we now request the assembly status to check the
         // *actual* status.
-        const resp2 = await client.getAssembly(id)
-        expect(resp2.ok).toBe('ASSEMBLY_CANCELED')
+
+        const timeout = 30000
+        const startedAt = new Date()
+        let lastStatus
+        for (;;) {
```

Comment:
out of curiousity, why not `while (true)`?

Reply:
no good reason ^^ I just think that eslint has complaining about `while(true)` but not `for(;;)` so it became a habit

Reply:
I see :D I've added the eslint comments for that but wasn't happy about it either.